sys/net/gnrc/link_layer/lorawan/include/gnrc_lorawan_internal.h

```diff
@@ -99,6 +99,14 @@ extern "C" {
 #define GNRC_LORAWAN_FOPTS_MAX_SIZE (15U)               /**< Maximum size of Fopts field */
 #define GNRC_LORAWAN_FPORT_SIZE (1U)                    /**< Size of the Fport field */
 
+#if defined(CPU_FAM_STM32F2) || defined(CPU_FAM_STM32F4) || \
```

Comment:
what's the rationale of this? Why do you treat these platforms differently?

Reply:
These CPU's have different flashpage sizes. `FLASHPAGE_NUMOF` does not account for this and therefore using `FLASHPAGE_NUMOF -0x4` would not work since that flashpage doesn't exist.

Reply:
How many flash pages do you need?

Reply:
1

Reply:
So why not always use the last one?

Reply:
How ? Using `flashpage_last_free` ?

Reply:
Why not use `FLASH_WRITABLE_INIT`?

Reply:
Because it requires the `periph_flashpage_in_address_space` feature which would prevent boards like the `nucleo-f411re` from being able to use LoRaWAN 1.1